### ISSUE TITLE
Enhanced accessibility features

### ISSUE THREAD

**Issue by javieralaves** [AutarkLabs/product#18]
## Description

Addressing accessibility issues found by the react-axe audits on Open Enterprise apps.

### Acceptance criteria

- [ ] Update non-aragonUI components with recognized issues identified by the audit
- [ ] AragonUI components with recognized issues have been identified
- [ ] Suggested changes per component have been documented and delivered to the Product team
- [ ] Suggestions have been communicated and discussed with Aragon One
- [ ] Suggestions have been requested on aragonUI for components to be updated
- [ ] Components have been updated with improved accessibility standards

**Comment by Quazia** [comment]
For a lot of these accessibility features we'll need to make changes to aragon-ui components. Do we want to cover updating the aragon-ui comonents in this epic, or would we want to make our own versions of the components?

**Comment by stellarmagnet** [comment]
I don't think we should maintain our own version of the aragonUI components. I think if it requires changing an actual aragonUI component, the deliverable for those use cases can involve cover clearly specifying the changes that need to be made to the components, so we can discuss it with Aragon One.

**Comment by javieralaves** [comment]
@Quazia @stellarmagnet I've added some acceptance criteria to this epic to account for the above. Let me know if this makes sense.

**Comment by stellarmagnet** [comment]
I like the updates @javieralaves! We need to add that non-aragonUI components should be updated. We may want to create another epic for actually updating the aragonUI components.

**Comment by javieralaves** [comment]
@stellarmagnet Alternatively, could we fit it an 'Update aragonUI components' issue on to epic #22 'Maintaining Open Enterprise'?

**Comment by stellarmagnet** [comment]
@javieralaves that would be something that comes after we code the updates to the aragonUI components and get them merged into the aragon/aragonUI repo. If we are having epics based on initiatives, both are related to making Aragon more inclusive.

Regardless, the changes to Open enterprise may not be possible until January or February, if you take into account the review cycles for aragonSDK

**Comment by javieralaves** [comment]
As far as this epic goes, shall we focus then first on non-aragonUI components (I've updated the acceptance criteria to reflect that), and then if there are any aragonUI components that need changing; let Radek document those for us so we can get Aragon One involved as soon as possible?

PS: If any of our non-aragonUI components don't match accessibility standards, also let me know so we can work it out on Figma in parallel.
